Maya/AETemplates/AEbabylonAiStandardSurfaceMaterialNodeTemplate.mel

```diff
@@ -0,0 +1,27 @@
+global proc AEbabylonAiStandardSurfaceMaterialNodeTemplate ( string $nodeName )
```

Comment:
(This comment applies to 3ds Max/Samples/sourceModels/BabylonMaterialAttributes/BackfaceCulling.max)

![image](https://user-images.githubusercontent.com/3952729/68173334-81ef4c00-ff2f-11e9-8b27-43f38694ebbe.png)

I've noticed that this scene raises a corruption warning? Can you look into what may be causing this?

Reply:
It appears related to these scripts possibly being referenced by the scene? They do not exist on my local computer, however...

![image](https://user-images.githubusercontent.com/3952729/68173582-46a14d00-ff30-11e9-8b85-9edc7c224317.png)